src/string/mod.rs

```diff
@@ -229,9 +232,11 @@ impl<const N: usize> String<N> {
     /// # Ok::<(), core::str::Utf8Error>(())
     /// ```
     #[inline]
-    pub fn from_utf8(vec: Vec<u8, N>) -> Result<Self, Utf8Error> {
+    pub fn from_utf8<LenT: LenType>(vec: Vec<u8, N, LenT>) -> Result<Self, Utf8Error> {
```

Comment:
This can just use `usize`, unless you plan on also making `StringInner` generic.

Reply:
Ah, nevermind, the `Vec` can be generic even if `StringInner` isn't.